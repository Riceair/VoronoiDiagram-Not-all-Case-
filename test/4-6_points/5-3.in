5
567 234
79 34
34 90
432 453
77 111