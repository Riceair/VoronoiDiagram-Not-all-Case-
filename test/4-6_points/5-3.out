P 34 90
P 77 111
P 79 34
P 432 453
P 567 234
E -228457 251011.5 -328.86 887.53
E -35210.71 -28277.73 69.29 72.27
E -328.86 887.53 69.29 72.27
E -328.86 887.53 308.62 225.84
E 69.29 72.27 345.37 79.44
E 308.62 225.84 345.37 79.44
E 308.62 225.84 138469.5 85393.5
E 345.37 79.44 126323 -307306
